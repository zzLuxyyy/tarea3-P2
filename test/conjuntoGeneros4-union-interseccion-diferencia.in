crearConjuntoGeneros 10
insertarConjuntoGeneros 0
insertarConjuntoGeneros 1
insertarConjuntoGeneros 2
insertarConjuntoGeneros 3
insertarConjuntoGeneros 6
insertarConjuntoGeneros 7
insertarConjuntoGeneros 8
insertarConjuntoGeneros 9
imprimirConjuntoGeneros
# unión con 1
unionConjuntoGeneros 1 1
# intersección con 1
interseccionConjuntoGeneros 1 1
# diferencia con 1
diferenciaConjuntoGeneros 1 1
# intersección con 1 3 4 5 6
interseccionConjuntoGeneros 5 1 3 4 5 6
# diferencia con 1 3 4 5 6
diferenciaConjuntoGeneros 5 1 3 4 5 6
# unión con conjunto vacío
unionConjuntoGeneros 0
# intersección con conjunto vacío
interseccionConjuntoGeneros 0
# diferencia con conjunto vacío
diferenciaConjuntoGeneros 0
liberarConjuntoGeneros
Fin