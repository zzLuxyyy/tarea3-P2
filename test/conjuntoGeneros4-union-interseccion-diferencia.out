1>Se creó conjunto generos con cantMax 10 exitosamente.
2>Se ejecutó 'insertarTConjuntoGeneros con genero de id 0 exitosamente.
3>Se ejecutó 'insertarTConjuntoGeneros con genero de id 1 exitosamente.
4>Se ejecutó 'insertarTConjuntoGeneros con genero de id 2 exitosamente.
5>Se ejecutó 'insertarTConjuntoGeneros con genero de id 3 exitosamente.
6>Se ejecutó 'insertarTConjuntoGeneros con genero de id 6 exitosamente.
7>Se ejecutó 'insertarTConjuntoGeneros con genero de id 7 exitosamente.
8>Se ejecutó 'insertarTConjuntoGeneros con genero de id 8 exitosamente.
9>Se ejecutó 'insertarTConjuntoGeneros con genero de id 9 exitosamente.
10>Conjunto de generos:
0 1 2 3 6 7 8 9
11>#  unión con 1.
12>Conjunto de generos unión:
0 1 2 3 6 7 8 9
13>#  intersección con 1.
14>Conjunto de generos intersección:
1
15>#  diferencia con 1.
16>Conjunto de generos diferencia:
0 2 3 6 7 8 9
17>#  intersección con 1 3 4 5 6.
18>Conjunto de generos intersección:
1 3 6
19>#  diferencia con 1 3 4 5 6.
20>Conjunto de generos diferencia:
0 2 7 8 9
21>#  unión con conjunto vacío.
22>Conjunto de generos unión:
0 1 2 3 6 7 8 9
23>#  intersección con conjunto vacío.
24>Conjunto de generos intersección:

25>#  diferencia con conjunto vacío.
26>Conjunto de generos diferencia:
0 1 2 3 6 7 8 9
27>Se ejecutó 'liberarTConjuntoGeneros' exitosamente.
28>Fin.
